### PR TITLE
script_output() - Add code comment for commit 72efc09b

### PR DESCRIPTION
**Pull request by cfconrad**
@okurz WDYT?

### CODE REVIEW

File: distribution.pm [hunk 1]
```diff
@@ -239,6 +239,9 @@ sub script_output {
         bmwqemu::log_call("Content of $script_path :\n \"$cat\" \n");
         testapi::type_string($cat . "\n");
         testapi::wait_serial("$cat", no_regex => 1, quiet => $args{quiet});
+        # Wait for input prompt of here tag before typing $script. This avoid
```
Comment (okurz) [inline distribution.pm]: avoid**s**

Comment (cfconrad) [reply distribution.pm]: thx

**Comment by okurz** [comment]
Adding the comment unfortunately demands a style change now, see https://travis-ci.org/os-autoinst/os-autoinst/builds/604891540#L839

**Comment by cfconrad** [comment]
thx for the pointer. Somehow I overseen it.

**Comment by okurz** [comment]
nice, thanks!

**Pull request merged**